grakn-graph/src/test/java/ai/grakn/graph/internal/ValidatorTest.java

```diff
@@ -129,20 +130,12 @@ public void whenCommittingNonAbstractRelationTypeNotLinkedToAnyRoleType_Throw(){
     @Test
     public void whenCommittingAbstractTypeWithInstances_Throw(){
         EntityType x1 = graknGraph.putEntityType("x1");
-        graknGraph.putEntityType("x2");
-        EntityType x3 = graknGraph.putEntityType("x3");
-        EntityType x4 = graknGraph.putEntityType("x4");
         x1.addEntity();
 
-        x1.setAbstract(true);
-        x4.setAbstract(true);
-
-        x4.superType(x3);
-
-        expectedException.expect(GraknValidationException.class);
-        expectedException.expectMessage(containsString(ErrorMessage.VALIDATION_IS_ABSTRACT.getMessage(x1.getLabel())));
+        expectedException.expect(GraphRuntimeException.class);
+        expectedException.expectMessage(containsString(ErrorMessage.IS_ABSTRACT.getMessage(x1.getLabel())));
 
-        graknGraph.commit();
+        x1.setAbstract(true);
```

Comment:
I'd delete this test, it seems the same as the above. The test name is also now incorrect.